tests/test_ivp_3D_simulation.py

```diff
@@ -70,13 +70,16 @@ def test_ivp_3D_simulation():
     sensor = kSensor(sensor_mask)
 
     # input arguments
-    input_filename  = f'example_input.h5'
-    input_file_full_path = os.path.join(pathname, input_filename)
+    input_filename = f'example_ivp_3D'
```

Comment:
Same here?

Reply:
addressed in [d876b13](https://github.com/waltsims/k-wave-python/pull/64/commits/d876b13e871b4a1860f672ecc3dd402383e218e9)